src/aosm/azext_aosm/generate_nfd/cnf_nfd_generator.py

```diff
@@ -698,8 +695,9 @@ def get_chart_name_and_version(
 
         if not os.path.exists(chart):
             raise InvalidTemplateError(
-                f"There is no Chart.yaml file in the helm package '{helm_package.name}'. \
-                    Please fix this and run the command again."
+                "There is no Chart.yaml file in the helm package"
+                f" '{helm_package.name}'.                     Please fix this and run"
```

Comment:
And again

Reply:
Sorry, and thanks for catching. I think that was the merge from main messing things up :-(